tests/unit/models/test_projects.py

```diff
@@ -56,6 +56,8 @@ def _project_yaml_data(
             "grade": "stable",
             "confinement": "strict",
             "parts": {},
+            "contact": "hello@world",
```

Comment:
I would drop these from this yaml because this fixture contains the bare minimum fields for a project and is used to test default values.

Instead, can you add new tests?

It looks like this is implicitly tested in `tests/unit/parts/test_update_metadata.py` but it's good practice to test it explicitly here. I'm thinking something like this:

```python
class TestProjectValidation:
    """Validate top-level project items."""

    ...

    def test_links_scalar(self, project_yaml_data):
        data = project_yaml_data(
            contact="hello@world",
            # set other links...
        )
        project = Project.unmarshal(data)
        assert project.contact == ["hello@world"]
        # assert other links
        
    def test_links_list(self, project_yaml_data):
        data = project_yaml_data(
            contact=["hello@world"],
            # set other links...
        )
        project = Project.unmarshal(data)
        assert project.contact == ["hello@world"]
        # assert other links
```